package.json

```diff
@@ -29,19 +29,15 @@
   "author": "",
   "license": "MIT",
   "peerDependencies": {
-    "react-dom": "^0.14.7 || 15.x || 16.x || 17.x",
```

Comment:
I can't think of an easy way to maintain support for older versions of React. Since this library generates its own root node, I think you'll have to drop support for the older versions.